src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs

```diff
@@ -4889,6 +4899,12 @@ public sealed class EnablePSRemotingCommand : PSCmdlet
             if ($dotPos -ne -1) {{
                 $powershellVersionMajor = $powershellVersionMajor.Substring(0, $dotPos)
             }}
+            # If we are running a Preview version, we don't want to clobber the generic PowerShell.6 endpoint
+            # but instead creating a PowerShell.6-Preview endpoint
+            if ($PSVersionTable.PSVersion.ToString().Contains(""preview""))
+            {{
+                $powershellVersionMajor += ""-Preview""
```

Comment:
Minor note: I'm curious why we don't use the same casing for Preview as we do in the version string?  In one case, we have mixed case Preview (PowerShell.6-Preview) while in the other, lower case (PowerShell.6.1.0-preview.715)

Reply:
semver.org uses lowercase for their examples of `alpha` and `beta` releases, but I don't see any language indicating it must be lowercase.  We should consider having a capital `P` in the semver and package name, but that's outside the scope of this PR.

Reply:
capitals are not allowed in many semver.  I've had to add ToLower to many  places in the code to ensure we can release without error.